package.json

```diff
@@ -69,6 +70,7 @@
     "eslint-plugin-react": "^6.10.2",
     "eslint": "^3.14.0",
     "extract-text-webpack-plugin": "~2.1.0",
+    "fs-extra": "^3.0.1",
```

Comment:
@reaktivo specifying a different `dist` folder? Hmmm, this is the folder used for the 'deliverable' of the package. It won't be built anymore on rwd :) Only copied ;) :)